### DISCUSSION TITLE
CJK and Russian words in Text to Speech

### DISCUSSION THREAD

**Discussion by VijayShankar1** [edrlab/thorium-reader#1809] [General]
Hi Team,

The Chinese/Korean or Russian words in Text to Speech(TTS) or Read Aloud features are NOT working in version 2.1.0 for Window 10 Pro. The CJK or Russian words are skipping in Read Aloud and the TTS is working for English and other European Languages. We are already provided the correct language attribute codes.

However the same Chinese/Korean words Text to Speech(TTS) features is working fine in version 2.1.0 for iMac system X (El Caption version).

Please suggest.

**Comment by danielweck** [comment]
hello, thank you for reporting this. we are aware of voice / language selection problems in Windows.
in thorium's toolbar at the top, there is a voice selector. what is the available selection? can you see voices that match the book language? if not, then you must make sure that suitable voices are installed in the operating system. you must also verify that the voices are exposed by Microsoft Windows to the app (there is a registry hack that reveals voices which are normally hidden by the vendor). I know it sounds complicated, but I am afraid this is out of our control.

**Reply by VijayShankar1** [reply]
Many thanks Daniel!

After installing the Voice packages from Microsoft Windows, the TTS (Read Aloud) features is working fine for CJK Languages.
